app/script/event/EventRepository.js

```diff
@@ -698,6 +698,7 @@ z.event.EventRepository = class EventRepository {
       event.server_time = event.time;
       event.time = storedEvent.time;
       event.primary_key = storedEvent.primary_key;
+      event.category = z.message.MessageCategorization.categoryFromEvent(event);
```

Comment:
@gregor I hesitated to put this logic in the `update_event` method of the `conversationService` but not sure if it makes sense in all the cases. WDYT?

Reply:
LGTM